log/context.go

```diff
@@ -23,49 +44,72 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var (
-	// G is an alias for GetLogger.
-	//
-	// We may want to define this locally to a package to get package tagged log
-	// messages.
-	G = GetLogger
+// L is an alias for the standard logger.
+var L = logrus.NewEntry(logrus.StandardLogger())
 
-	// L is an alias for the standard logger.
-	L = logrus.NewEntry(logrus.StandardLogger())
-)
+type loggerKey struct{}
 
-type (
-	loggerKey struct{}
+// Fields type to pass to "WithFields".
+type Fields = map[string]any
 
-	// Fields type to pass to `WithFields`, alias from `logrus`.
-	Fields = logrus.Fields
+// Entry is a logging entry. It contains all the fields passed with
+// [Entry.WithFields]. It's finally logged when Trace, Debug, Info, Warn,
+// Error, Fatal or Panic is called on it. These objects can be reused and
+// passed around as much as you wish to avoid field duplication.
+//
+// Entry is a transitional type, and currently an alias for [logrus.Entry].
+type Entry = logrus.Entry
```

Comment:
Entry is the tricky one; I tried if I could replace it with an interface, but unfortunately that's not (directly) possible, as some code depends on exported fields in the type.

We may be able to swap out things once all code (including plugins etc) have transitioned and directly depend on this package.